docs/admin/serverless/servicemesh/README.md

```diff
@@ -263,4 +295,123 @@ kubectl exec -it sleep-6d6b49d8b8-6ths6 -- curl -v sklearn-iris-burst-predictor-
     <
     * Connection #0 to host sklearn-iris-burst-predictor-default.user1.svc.cluster.local left intact
     {"name":"sklearn-iris-burst","ready":true}
-    ```
\ No newline at end of file
+    ```
+
+## Invoking InferenceServices from workloads that are not part of the mesh {#invoking-isvc-non-mesh}
+Ideally, when using service mesh, all involved workloads should belong to the service mesh. This allows enabling strict TLS in Istio and ensures policies are correctly applied. However, given the diverse requirements of applications, it is not always possible to migrate all workloads to the service mesh.
```

Comment:
`all involved workloads should belong to the service mesh`

to the same service mesh?

Reply:
AFAIK, upstream Istio is a "singleton" on a cluster.